pkg/build/Dockerfile.template

```diff
@@ -34,12 +39,21 @@ RUN cd ${PLAN_DIR} \
     && go env -w GOPROXY=direct \
     && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o testplan ${TESTPLAN_EXEC_PKG}
 
+# Optionally install IPFS
+RUN if [ -n "${GO_IPFS_VERSION}" ]; then echo Install IPFS ${GO_IPFS_VERSION}; bash /install-ipfs.sh ${GO_IPFS_VERSION}; fi
```

Comment:
That was my first attempt ... I was trying to keep a single Dockerfile.template that worked for both the DHT test and the smlbench test - Dockerfile doesn't support conditionals like "if" statements, so I needed to have a shell script for that. I'm not really happy with how it turned out, so I think it might make more sense to have multiple templates, or a template that is generated dynamically by testground.